### DISCUSSION TITLE
Comfort of work in Godot Editor (Android) - Updated English

### DISCUSSION THREAD

**Discussion by DualFidexGames** [godotengine/godot-proposals#9702] [Platforms]
The Android Godot Editor doesn't feel as comfortable as the PC one. Or rather, like a mobile apk. I propose that more time be dedicated to making Godot Editor (Android) feel as comfortable as the one on PC, but stable like the one on mobile.

I will report the bugs in the bugs section. But it must be said that they make the mobile version stable and comfortable. It would be better for those people who have their PCs at home and are far away but with a mobile phone.

This way, when the developer uploads the project to GitHub, he can continue it on the beach, hotel, countryside, or anywhere he has time to continue working on his mobile.

Eye! Spend time on the mobile version of Godot Editor. It will make users, including those who do not have a PC. They can work comfortably.

Bug report:

The bugs in the mobile version are the following:

1- The project debug window causes me to close the project when there is a code syntax error: I know that on mobile when running the debug window, it cannot be opened like another application (I imagine due to Android limitations).

But it would be nice if when you make an error that the game crashes, I simply close the project's debug screen and let you go to the editor and see what the error is (Since you can't see why I know Saw the project)

2- The manual work of the Script editor is not as comfortable as that of a normal mobile: the Script editor does not feel like a mobile editor. Rather, he acts as if he will try to be like PC. For example, when dialing a code. You have to hold your finger down outside the script and automatically move it to the top of the code to select it.

I propose that the editor feel like a mobile text editor. Since mobile users will feel more comfortable with the editor.

3- the 3D editor is very slow to scroll through the world: I'm not sure if there is a setting to change the sensitivity of the 3D editor joysticks

But it would be nice if it had a setting to change it. Because I notice it very slow.

4- Joysticks from the 3D editor disappear when I imported a 3D project already made on PC: when I imported a Godot Engine project from PC to mobile... The joysticks disappeared and I couldn't move. I don't know if it's because they took a while to load, or because it's too much of a load for my mobile device.

I propose that I know I can fix that.

These errors and more would be better if they were solved in the Godot Mobile Editor.

PS: my phone is a Samsung A03 Core

**Comment by Calinou** [comment]
This is an English-speaking platform, please write in English (use machine translation if needed).

> 3- the 3D editor is very slow to scroll through the world: I'm not sure if there is a setting to change the sensitivity of the 3D editor's joysticks

Yes, you can increase **Editors > 3D > Freelook > Freelook Sensitivity** in the Editor Settings. **Freelook Base Speed** controls the movement speed.

**Reply by DualFidexGames** [reply]
ok, tanks you

**Comment by theludovyc** [comment]
Hello :wave: 

Maybe you can be interested by this issue https://github.com/godotengine/godot-proposals/issues/10140 :blush:

**Reply by DualFidexGames** [reply]
It's a good topic to discuss... I also saw that they proposed the option of the editor in vertical mode...

For me the vertical mode is not so necessary, because the workspace would be limited... Unless the UI is simplified for the devices in that mode

Anyway, for me it is much more comfortable in horizontal mode... But yes, I see the editor as very unstable... We would have to see what can be done to make it more stable, and not imitate the PC version adapted to mobile... That is what I see it does

I know they do it so that it is easy to export it to mobile... But I think it is the best for the editor (As long as the file format is 100% compatible with the PC version). Yes, you could work on a PC at home/work and then on the street, not somewhere else, with your mobile.
